tests/unit/backend/corpora/common/entities/__init__.py

```diff
@@ -0,0 +1,2 @@
+def get_ids(iterable):
```

Comment:
I think I've made this comment before, but I'd prefer not to have utility functions like this hidden away in `__init__` files. Utility functions should be in utils.py or util.py or some file that clearly designates the utility status.

Also, I wonder what the value is of having a function that is composed of one line. For testing purposes, code repetition is fine, especially for one line, because it makes the test self-sufficient. I find self-sufficiency in tests to be more valuable that code reuse. I don't want to jumping around files to visit utility functions... how do I know that I haven't introduced a bug in my test via untested utility functions?

Reply:
it's used in 24 different places. It does simply the code significantly.